ADC_irq/Core/Src/main.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * 	Implement a routine that reads the internal voltage reference (ADC_IN17)
+ * 	and internal temperature sensor (ADC_IN18).
+ * 	The internal reference voltage must be in millivolts and the temperature in Celsius.
+ *
+ * 	1 - Polling		[X]
+ * 	2 - Interrupt	[/]
+ * 	3 - DMA			[ ]
+ */
+
+#include "stm32l053xx.h"
+#include "macros.h"
+#include "struct.h"
+
+ADC_Status	ADC_Config(uint32_t Channel);	//we call this function with the ADC channel we want to read
+void 		wait(uint16_t time);			//delay function, system clock based
+
+uint8_t	flag_EOC = 0;						//flag for End Of Conversion (ADC converted one single channel)
+
+void ADC1_COMP_IRQHandler(void)
+{
+	if(ADC1->ISR & ADC_ISR_EOC)			//if there was and End Of Conversion

```

Comment:
Shouldn't it be "an" instead of "and"?

Reply:
yes, misspelling...

about the tabulation,
![Screen Shot 2020-04-24 at 14 55 50](https://user-images.githubusercontent.com/61191550/80242413-b1a82280-863b-11ea-9e7c-b8c435f9d996.png)